ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -98,24 +98,19 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-aws-ipi
-- as: e2e-aws-ipi-hypershift
+- as: e2e-aws-ipi-ovn-hypershift-mgmt-p2
   cron: 0 4 */7 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
-      TAG_VERSION: '@4.11'
-      TEST_SCENARIOS: Hypershift
-    post:
-    - chain: hypershift-aws-destroy
-    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
-    pre:
-    - chain: cucushift-installer-rehearse-aws-ipi-provision
-    - chain: hypershift-install
-    - chain: hypershift-aws-create
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftMGMT&;~Disruptive&
+      TEST_SCENARIOS: MCO|ETCD|PSAP|SDN|STORAGE|API_Server|Authentication|Cluster_Operator
```

Comment:
@LiangquanLi930 please use TEST_ADDITIONAL to set it as Hypershift, and do not set TEST_FILTERS to let it as default. it will let you only focus on hypershift, and take general value for other subs. it is easy to maintain it later.